cobalt/browser/application.cc

```diff
@@ -1021,6 +1021,8 @@ Application::Application(const base::Closure& quit_closure, bool should_preload,
         base::TimeDelta::FromSeconds(duration_in_seconds));
   }
 #endif  // ENABLE_DEBUG_COMMAND_LINE_SWITCHES
+
+  AddCrashLogApplicationState(base::kApplicationStateStarted);
```

Comment:
My only mild concern here is a race condition where the crash extensions aren't ready to accept this value. Should the Starboard extensions initialized at this point already?

Reply:
The logic in `AddCrashLogApplicationState()` https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:cobalt/browser/application.cc;l=597 attempts to get the extension and falls back to the crash log dictionary if it is not available.

I'm also assuming that starboard extensions are already initialized at this point because this logic I'm adding is in line 1025 of `Application()` which is after a similar existing call in line 891 (https://source.corp.google.com/h/lbshell-internal/cobalt_src/+/COBALT:cobalt/browser/application.cc;l=891).